examples/tensorflow_converter/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required( VERSION 3.2 )
+
+project( tensorflow_converter )
+
+find_package( Protobuf 3.0.0 REQUIRED )
+
+file(GLOB PROTO_FILES ${CMAKE_CURRENT_SOURCE_DIR}/../../tiny_dnn/io/tensorflow/tensorflow/core/framework/*.proto)
+
+# generate CPP file for connexion between protobuf and tiny_dnn 
+PROTOBUF_GENERATE_CPP( PROTO_SRC PROTO_HDR ${PROTO_FILES} )
```

Comment:
I'll check it as soon as possible.

But usually try to avoid GLOB in cmake, it's a really bad idea (even though it's simple), the best is to define a variable containing exactly the files you want.